guides/govuk-paas.md

```diff
@@ -0,0 +1,42 @@
+---
+category: Development Guides
+expires: 2020-05-01
+---
+
+# GOV.UK Platform as a Service
+
+Gov.uk Platform as a Service (PaaS) is a platform developed by [Government Digital Service (GDS)](https://www.gov.uk/government/organisations/government-digital-service) and available for Government departments to deploy applications and backend services.
+
+## Suitability
+
+It simplifies deploying to the cloud as a lot of the requirement for production applications is already implemented: internet access, routing, container scheduling, scalability, observability, redundancy, logging, monitoring, user access management, billing...
+
+It works well for a number of use cases. When more customisation is required, it is recommended to use [Azure CIP](/guides/default-technology-stack/#azure).
+
+### 12-factor apps
+PaaS is mainly aimed at [12-factor applications](https://12factor.net/), which is a set of good practices for web applications to ensure smooth operations, scalability and consistency across environment.
+
+### Security
+The platform is assured for use at OFFICIAL. As the data owner for a service [you must ensure that appropriate controls are in place when handling Official data](https://www.gov.uk/guidance/official-sensitive-data-and-it).
+
+### Containers
+It runs applications in stateless containers, horizontally scalable. It is not adapted to large stateful entreprise applications.
+
+### Internet facing
+The platform runs in the cloud and is exclusively accessed via Internet for inbound and outbound connections. It can't be connected via dedicated network links or use IP restrictions.
+
+### Backing services
+A limited number of backing services are provided: postgres, mysql, redis, elasticsearch, AWS S3, InfluxDB. They run in the platform and cannot be accessed from outside.
```

Comment:
Can link to this page here too: https://www.cloud.service.gov.uk/application-development/#backing-services